elastic/apm-server
# apmpackage: Add back "message" to error_logs ds

## Motivation/summary

Adds back the ecs base "message" field to as an external `ecs` field to
the `error_logs` data stream.

It causes the errors captured by APM to not have a `message` field which
in turn causes the Logs tab to not display the error message.

## How to test these changes

1. Install the APM Integration
2. Save this program as `main.go` and run with `go run main.go`

```go
package main

import (
	"context"
	"errors"
	"log"

	"go.elastic.co/apm"
)

func main() {
	tx := apm.DefaultTracer.StartTransaction("apm-server-7.16.0", "test-plan")
	ctx := apm.ContextWithTransaction(context.Background(), tx)
	span, ctx := apm.StartSpan(ctx, "capture-error", "type")
	err := apm.CaptureError(ctx, errors.New("a terrible error"))
	err.Send()
	span.End()
	tx.End()
	apm.DefaultTracer.Flush(nil)
	log.Printf("sent %+v\n", apm.DefaultTracer.Stats())
	log.Println("done")
}
```
3. Navigate to the and verify that an error is ingested and under the `Logs` tab `"a terrible error"` is displayed:

<img width="1703" alt="Screen Shot 2021-10-28 at 4 58 53 PM" src="https://user-images.githubusercontent.com/7286993/139223370-db5772a8-672e-49e6-b3b2-6d14169ea46b.png">

## Related issues

This bug was introduced by #6359.
Discovered while testing #5974
